contracts/OffsetHelper.sol

```diff
@@ -411,10 +407,6 @@ contract OffsetHelper is OffsetHelperStorage {
             balances[msg.sender][_tco2s[i]] -= _amounts[i];
 
             IToucanCarbonOffsets(_tco2s[i]).retire(_amounts[i]);
-
```

Comment:
we can move the i++ to unchecked in for loop, and remove the initializer for i to optimize for gas

Reply:
but I am not fully removing the i . So how can I remove the inititalizer?

Reply:
as in you can keep it as uint256 i; instead of uint256 i=0; as it is already defined to zero by default.